### PR TITLE
Fix a problem with 'Javadoc' task

### PR DESCRIPTION
**Pull request by Oknesif**
Had a problem running 'build' task for the project. The problem was in 'Javadoc' task, so I added 'source' in the configuration of the task and changed links to 'https' instead of 'http'.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -221,14 +221,15 @@ tasks.withType(Test) {
 tasks.withType(Javadoc) {
     title = "${project.name} ${project.version}"
     configure(options) {
+        source = JavaVersion.VERSION_1_8
```
Comment (benjamin-bader) [inline build.gradle:224]: What is this meant to do?  `source` on Javadoc tasks should be files, but here we're passing in a version string.  Why?

**Comment by benjamin-bader** [comment]
Thanks for contributing!  Would you please elaborate on what the problem actually was?  I can't tell at all what the `source` change does or is supposed to do, and would appreciate an explanation.

File: build.gradle (hunk 1 above)
Comment (benjamin-bader) [reply build.gradle:224]: OK, found it - Gradle docs are hard to navigate sometimes: https://docs.gradle.org/current/javadoc/org/gradle/external/javadoc/CoreJavadocOptions.html#getSource--

tl;dr this sets the language version allowable by the javadoc process.

**Comment by benjamin-bader** [comment]
Alright, found the relevant docs.  This all seems fine.  Running CI workflow now, will merge if it passes.  Thanks!
